artview/components/plot_radar.py

```diff
@@ -586,10 +587,10 @@ def getPathInteriorValues(self, path):
 
         Returns
         -------
-        x, y, azi, range, value, ray_idx, range_inx: ndarray
-            Truplet of 1arrays containing x,y coordinate, azimuth,
-            range, current field value, ray index and range index
-            for all bin of the current radar and tilt inside path.
+        points: Points
+            Points object containing all bins of the current radar
+            and tilt inside path. Axes : 'x_disp', 'y_disp', 'ray_index',
+            'range_index', 'azimuth', 'range'. Fields: just current field
 
```

Comment:
I like the fact that you included the field in here. I had trouble figuring out how to do this.